openfisca_france/model/prestations/prestations_familiales/af.py

```diff
@@ -103,7 +103,7 @@ def formula(famille, period, parameters):
         plus_de_deux_enfants = (af_nbenf >= 2) * pfam.taux.enf2
         plus_de_trois_enfants = max_(af_nbenf - 2, 0) * pfam.taux.enf3
         taux_total = un_seul_enfant + plus_de_deux_enfants + plus_de_trois_enfants
-        montant_base = eligibilite * round(pfam.bmaf * taux_total, 2)
+        montant_base = eligibilite * round_(pfam.bmaf * taux_total, 2)
```

Comment:
> What is the difference between round() and round_() ?

Reply:
[`round_`](http://openfisca.org/doc/coding-the-legislation/25_vectorial_computing.html#forbidden-operations-and-alternatives) is the numpy version that applies to vectors.

It is part of the `model_api` and automatically imported in variable files. Here, it was manually imported as `round`, which is not the standard.